src/System.Security.Cryptography.Cng/src/Internal/Windows/Crypt32/Interop.Crypt32.HashIdAlg.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.NativeCrypto;
+using System;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+
+internal partial class Interop
+{
+    internal static partial class Crypt32
+    {
+        private const string Capi2Dll = "Crypt32.dll";
```

Comment:
Don't we already have this defined in Libraries?  If not, can we add it there instead and then use that in the DllImport?